Cipher.js

```diff
@@ -188,6 +188,46 @@ export class Cipher {
     return JSON.parse(new TextDecoder().decode(data));
   }
 
+  /**
+   * Creates a JOSE Header and encrypted_key for a recipient.
+   *
+   * @see https://tools.ietf.org/html/rfc7516#section-4
+   *
+   * @param {object} options - Options to use.
+   * @param {object} options.recipient - A recipient with a header with a
+   *   kid and alg.
+   * @param {object} options.ephemeralKeyPair - An ephmerealKeyPair.
+   * @param {object} options.cek - A content encryption key.
+   * @param {Function} options.keyResolver - A function that can resolve keys.
+   *
+   * @returns {Promise<object>} A JWE recipient object.
+   */
+  async addRecipient({recipient, ephemeralKeyPair, cek, keyResolver}) {
+    // ensure all recipients use the supported key agreement algorithm
```

Comment:
This comment doesn't make sense here. It's copied from somewhere else where the algorithm was being checked. It looks like that other place is unchanged by this PR, so this comment should be changed to:

```suggestion
    // resolve public DH key for recipient
```